orange/src/main/java/com/hartwig/hmftools/orange/OrangeApplication.java

```diff
@@ -32,17 +29,17 @@ public static void main(String[] args) throws IOException
         new OrangeApplication(config).run();
     }
 
-    @NotNull
     private final OrangeConfig config;
 
-    private OrangeApplication(@NotNull final OrangeConfig config)
```

Comment:
Why is `@NotNull` dropped here?

Reply:
We have adopted the convention of using @NotNull sparingly in hmftools to avoid so many annotations. But since Orange has it everywhere I've restored it in this instance.